composer.json

```diff
@@ -6,11 +6,11 @@
     "require": {
         "cweagans/composer-patches": "^1.6.4",
         "drupal-composer/drupal-scaffold": "^2.0.0",
-        "drupal/lightning_api": "3.x-dev",
-        "drupal/lightning_core": "4.x-dev",
-        "drupal/lightning_layout": "1.x-dev",
-        "drupal/lightning_media": "3.x-dev",
-        "drupal/lightning_workflow": "3.x-dev",
+        "drupal/lightning_api": "^3.5",
+        "drupal/lightning_core": "^4.0-alpha1",
+        "drupal/lightning_layout": "^1.6",
+        "drupal/lightning_media": "^3.7",
+        "drupal/lightning_workflow": "^3.4",
```

Comment:
I don't believe Lightning Workflow 3.4 allows Lightning Core 4.x. This constraint will allow it, but we should be more explicit (`"drupal/lightning_workflow": "^3.5"`)